### DISCUSSION TITLE
Creatives In Recovery Meeting

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/12-step-meeting-list#1530] [Q&A]
Hi Team, 

QQ for ya. We have a meeting in Covington KY, Creatives in Recovery, that was deleted from AA Cincinnati 8/27 at 5:22 but is still showing up on the MG app. Can you please tell us who else has this listed so we can get it removed?
![image](https://github.com/user-attachments/assets/cef2bb73-79b5-4e94-a75c-5515f81fda5c)

**Comment by ATLCOTech** [comment]
Cancel that. I was wrong. It was deleted this moring. Closing this case.

**Reply by joshreisner** [reply]
ok sounds good. in general, you can just scroll down on the listing a little to see which service entity provided it
